packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.js

```diff
@@ -15,27 +25,43 @@ export default class QuanticRecentQueriesList extends LightningElement {
     expand
   }
 
-  /** @type {import("coveo").RecentQueriesState} */
-  @track state;
-
-  /** @type {string} */
-  @api label = this.labels.recentQueriesLabel;
-  /** @type {number} */
-  @api maxLength = 10;
-  /** @type {string} */
+  /**
+   * The ID of the engine instance with which to register.
+   * @api
+   * @type {string}
+   */
   @api engineId;
-  /** @type {boolean} */
+  /**
+   * The maximum number of queries to keep in the list.
+   * @api
+   * @type {number}
+   */
+  @api maxLength = 10;
+  /**
+   * The non-localized label for the component.
+   * @api
+   * @type {string}
+   */
+  @api label = this.labels.recentQueriesLabel;
+  /**
+   * Whether the component should be collapsed.
+   * @api
+   * @type {boolean}
+   * @defaultValue `false`
+   */
   @api get isCollapsed() {
     return this._isCollapsed;
   }
   set isCollapsed(collapsed) {
     this._isCollapsed = collapsed;
   }
-    
   /** @type {boolean} */
   _isCollapsed = false;
+
+  /** @type {RecentQueriesState} */
+  @track state;
   
-  /** @type {import("coveo").RecentQueriesList} */
+  /** @type {RecentQueriesList} */
   recentQueriesList;
   /** @type {()=> void} */
```

Comment:
Are you able to generate docs.json at the moment? I'm getting errors in the console, one of which mentions this line but I'm not sure what's wrong with it 🤔

1:
`ERROR: Unable to parse a tag's type expression for source file /Users/jpmarceaucoveo.com/Projects/ui-kit/packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js in line 122 with tag title "type" and text "{import("coveo").CategoryFacet}}": Invalid type expression "import("coveo").CategoryFacet": Expected "!", "=", "?", "[]", "|", or end of input but "(" found.

ERROR: Unable to parse a tag's type expression for source file /Users/jpmarceaucoveo.com/Projects/ui-kit/packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js in line 123 with tag title "type" and text "{import("coveo").CategoryFacet}}": Invalid type expression "import("coveo").CategoryFacet": Expected "!", "=", "?", "[]", "|", or end of input but "(" found.`

2:
`ERROR: Unable to parse a tag's type expression for source file /Users/jpmarceaucoveo.com/Projects/ui-kit/packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.js in line 66 with tag title "type" and text "{()=> void}": Invalid type expression "()=> void": Expected "!", "=", "?", or end of input but ">" found.

ERROR: Unable to parse a tag's type expression for source file /Users/jpmarceaucoveo.com/Projects/ui-kit/packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.js in line 67 with tag title "type" and text "{()=> void}": Invalid type expression "()=> void": Expected "!", "=", "?", or end of input but ">" found.`

Reply:
fixed